packages/storybook8/stories/Components/ControlBar/Buttons/Camera/Camera.story.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CameraButton, CameraButtonProps, DEFAULT_COMPONENT_ICONS } from '@azure/communication-react';
+import { initializeIcons, registerIcons } from '@fluentui/react';
+import React, { useState } from 'react';
+import { defaultControlsCameras } from '../../../../controlsUtils';
+
+initializeIcons();
```

Comment:
Do we want to do the Icons initialization inside the `.story.tsx`? Or inside `snippet.tsx`?

Reply:
Moved it to `snippet.tsx`, as we are keeping the `registerIcons` in snippets